netcreateorg/netcreate-itest
# update-node-v18: nvmrc now v18.18.2

This updates our official node target to v18.18.2, which is the latest version of the LTS v18 release. This is supported into **April 2025**. The version is stored in the `.nvmrc` file in the repo. 

When updating node versions, it's generally a good idea to run the following commands again:
```
npm ci
```
as the `package-lock.json` file that contains dependencies has changed. There should be no issues since we were already on the v18 branch (previously, v18.16.0). We are making this change across all projects.

## TO TEST

1. Pull the branch
2. Make sure you are using the Visual Studio project file: `netcreate-itest.code-workspace`. This is how you should be making changes to netcreate, as the code workspace file contains important configuration settings that are only active.
3. `npm ci` to update any library dependencies that have changed
4. run netcreate normally, report any weird issues or crashes. Describe what you expected to happen, and include **screenshots** of what you saw instead (identify where to look so)
